### DISCUSSION TITLE
Pipeline sync failure for Trello source with S3 avro destination

### DISCUSSION THREAD

**Discussion by WisamulHaque** [airbytehq/airbyte#34106] [General]
I have created a connection using the following:
Source: Trello 
Destination: S3 avro

Sync failure:

![image](https://github.com/airbytehq/airbyte/assets/106967803/6d7434c3-012a-4fc5-af23-6f3e3fdd94d0)
![image](https://github.com/airbytehq/airbyte/assets/106967803/8aab0a0d-1c1a-48e4-b61f-00e209e28229)

Logs are attached below:
[trello_avro_logs.txt](https://github.com/airbytehq/airbyte/files/13889825/trello_avro_logs.txt)

**Comment by sajarin** [comment]
hey @WisamulHaque 

does this connection work, if you use something other than S3 avro? Based on the logs, it seems like there is an error with converting a few different fields from the schema of the trello streams.

[a comment by an automated account is not shown]
